packages/vscode-extension/src/webview/components/Preview.tsx

```diff
@@ -116,8 +116,19 @@ type Props = {
 };
 
 function Preview({ isInspecting, setIsInspecting, zoomLevel, onZoomChanged }: Props) {
+  interface TouchPoint {
+    x: number;
+    y: number;
+  }
+
   const wrapperDivRef = useRef<HTMLDivElement>(null);
   const [isPressing, setIsPressing] = useState(false);
+  const [isMultiTouching, setIsMultiTouching] = useState(false);
+  const [isPanning, setIsPanning] = useState(false);
```

Comment:
At this point I kinda want to make it an enum (string based instead of direct TS enum) instead of using three independent states. If you want, you can experiment with it.

Reply:
Each of those states is responsible for different aspect:
- `isPressing` indicates whether the left mouse button is currently pressed.
- `isMultiTouching` indicates whether the Option button is being held, changing the visibility of multitouch markers.
- `isPanning` indicates whether the Shift button is being held; when the user stops pressing it, the isMultiTouching variable is used to determine whether multitouch markers were previously visible.

I tried to simplify it but keeping previous state of an enum would be necessary, so I'll omit adding that enum.